### PR TITLE
#1344 & #1345 feat: epic add popover for user informations

### PR DESCRIPTION
**Pull request by reyhanrumengan**

### CODE REVIEW

**Comment by reyhanrumengan** [comment]
in the popover, roles and skill loads after 1-2 seconds. How to improve this?

**Comment by LetsPoke** [comment]
1. why you pass down the userId and then load the user information again instead of passing down the whole user object?
2. the user should not be able to edit his/her name and email

File: hms-app/src/components/HeaderMenu.tsx [hunk 1]
```diff
@@ -187,6 +218,80 @@ export default function HeaderMenu({
...
+        <Popover.Dropdown>
+          <Text size='sm'>
+            Name: {user?.name}
+          </Text>
+
+          <Text size='sm'>
+          Email: {user?.username}
+          </Text>
+
+        {isLoading && (
```
[the remark this answers is not in the document]
Comment (LetsPoke) [reply hms-app/src/components/HeaderMenu.tsx:259]: if we dont need it anymore you can remove it

**Comment by LetsPoke** [comment]
remove name and mail from editForm 
pass down dbUser instead of id

**Review by LetsPoke — APPROVED** [review]

**Pull request merged**